pkgs/applications/window-managers/awesome/git.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, awesome, fetchFromGitHub, ... }:
+
+let
+  version = "3.6-git-g4636b11";  # "git describe: v3.5.2-398-g4636b11"
```

Comment:
IMHO, for packages that have proper tags, the output of git describe is a better version identifier than a date. The way I see it, dates should only be used for packages that lack tags/version numbers in the first place.

Good:
```
  version = "3.5.2-398-g4636b11";
```

Less good:
```
  version = "3.5.2-git-2015-03-07";
```

The problem with the latter is that it is ambiguous. It can either be a pre-release (nixpkgs uses this scheme, although I dislike it), or it can be seen as a snapshot sometime after 3.5.2, which is the correct interpretation in this case.

Anyway, what is the point of having "awesome-git" in nixpkgs? If the development branch of awesome is ready for wider audience, why not prod upstream to make a new release? Or at least a release candidate of some sort? The expression is pretty small, so it shouldn't be that difficult to do these kind of modifications locally instead of having to push it to upstream nixpkgs...?